### DISCUSSION TITLE
v2.2.2

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3460] [分享 / Share]
⚠ 此版本起, 对 Chromium 内核浏览器的版本要求提升至 88

✨新增
- `网址参数清理` 支持清理 `vd_source`, 之前装了临时插件的用户可以在更新后卸载相应的插件. (#3424)

🐛修复
- 修复 `bwp-video` (HEVC 编码) 下 PlayerAgent 的异常行为:
  - 修复倍速扩展的问题. (#3398, PR #3400 by [JLoeve](https://github.com/LonelySteve), PR #3401 by [imshixin](https://github.com/imshixin))
  - 修复开关灯的问题. (#3403, PR #3413 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- 修复 `简化评论区` 的样式失效, 以及评论区的操作菜单失效. (#3425, #3453)
- 修复动态详情页的动态相关功能失效.
- 修复 `videoChange` 部分情况下无法触发.
- 修复 `简化直播间` 的直播间皮肤部分情况下无法屏蔽.
- 修复搜索栏的建议在话题页面布局错乱. (#3383)
- 修复 `图片批量导出` 中的命名格式未能正确赋值. (#3326)
- 修复 `快速收起评论` 收起后再打开评论区失效的问题.

☕开发者相关
- 修复 Pull Request 中 ESLint 未报告部分代码问题.
- `loadFeatureCode` 支持从代码的返回值中得到结果.
- 删除 jsDelivr 相关的 GitHub Action.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.2.2'>v2.2.2</a>.</em>

**Comment by tama8o** [comment]
默认播放模式最近失效了
